packages/angular_devkit/build_angular/src/builders/dev-server/schema.json

```diff
@@ -106,6 +106,24 @@
       "type": "boolean",
       "description": "Force the development server to use the 'browser-esbuild' builder when building. This is a developer preview option for the esbuild-based build system.",
       "default": false
+    },
+    "prebundle": {
```

Comment:
Should this have a default of true?

Reply:
Could go either way on the default.  It's enabled by default when using the application builder but its not used with the browser builder.

I'll add the caching info.

Reply:
I added some usage warnings that rely on the raw option value being undefined to check whether the option was actually specified. To allow that to work there cannot be a default in the schema.